app/controllers/AuditController.scala

```diff
@@ -40,43 +40,67 @@ class AuditController @Inject() (implicit val env: Environment[User, SessionAuth
     *
     * @return
     */
-  def audit = UserAwareAction.async { implicit request =>
+  def audit(nextRegion: Option[String]) = UserAwareAction.async { implicit request =>
     val now = new DateTime(DateTimeZone.UTC)
     val timestamp: Timestamp = new Timestamp(now.getMillis)
     val ipAddress: String = request.remoteAddress
 
     request.identity match {
       case Some(user) =>
-        WebpageActivityTable.save(WebpageActivity(0, user.userId.toString, ipAddress, "Visit_Audit", timestamp))
 
         // Check and make sure that the user has been assigned to a region
         if (!UserCurrentRegionTable.isAssigned(user.userId)) {
+          // Note: This condition is never true because a region is assigned immediately after the user signs up
           UserCurrentRegionTable.assignRandomly(user.userId)
         }
 
         var region: Option[NamedRegion] = RegionTable.selectTheCurrentNamedRegion(user.userId)
 
+        nextRegion match {
+          case Some("easy") =>
+            // Assign an easy region is the query string has nextRegion=easy
+            UserCurrentRegionTable.assignEasyRegion(user.userId)
+            region = RegionTable.selectTheCurrentNamedRegion(user.userId)
+          case Some("regular") =>
+            // Assign an easy region is the query string has nextRegion=regular
+            UserCurrentRegionTable.assignNextRegion(user.userId)
+            region = RegionTable.selectTheCurrentNamedRegion(user.userId)
+          case _ =>
+            ;
```

Comment:
Maybe a string that is passed in that does not match either of those formats should log a warning or something.